lib/cmds/blockchain/blockchain.js

```diff
@@ -126,7 +126,7 @@ Blockchain.prototype.initChainAndGetAddress = function() {
 
 var BlockchainClient = function(blockchainConfig, client, env, isDev) {
   if (client === 'geth') {
```

Comment:
Need a todo to add in all the other clients in here or hope that this all gets taken out with Eric's transpiling. Nothing big, just something I noticed.